server/athenian/api/models/state/models.py

```diff
@@ -227,8 +227,31 @@ class ReleaseSetting(create_time_mixin(updated_at=True), Base):
     """Release matching rules per repo."""
 
     __tablename__ = "release_settings"
+    __table_args__ = (
+        UniqueConstraint(
+            "repo_id",
+            "logical_repo_name",
+            "account_id",
+            name="uc_release_settings_repo_id_logical_name_account",
+        ),
+    )
 
     repository = Column(String(), primary_key=True)
```

Comment:
```suggestion
```
We don't need it anymore. Given that we test our data migration on production DB, we shouldn't have problems.

Reply:
Side note: we should check if reposet-sync tries to do smth with our table, and adjust it accordingly. I guess, it may act on renames and removals.

Reply:
`repository` is still used (and updated), the plan was to stop using it and then remove the column on the next PR
For example here, I guess the Prefixer will be used to then query for repo_id:
https://github.com/athenianco/athenian-api/blob/master/server/athenian/api/internal/settings.py#L725

Reply:
After a quick check I think reposet-sync updates only repository_sets

Reply:
DEV-4931